tailnet/configmaps.go

```diff
@@ -570,3 +571,16 @@ func prefixesDifferent(a, b []netip.Prefix) bool {
 	}
 	return false
 }
+
+// derpMapStringer converts a DERPMap into a readable string for logging, since
+// it includes pointers that we want to know the contents of, not actual pointer
+// address.
+type derpMapStringer tailcfg.DERPMap
+
+func (d *derpMapStringer) String() string {
+	out, err := json.Marshal((*tailcfg.DERPMap)(d))
+	if err != nil {
+		return "!!!error marshaling DERPMap"
```

Comment:
Should also include the message, even though I doubt this will probably ever get hit in practice